static/videoList/videoItem.js

```diff
@@ -1,47 +1,137 @@
               align-items: center;
               justify-content: space-between;
-              padding: 1rem;
+              width: 100%;
+              background: #262626;
+              margin: 6px;
+              padding: 12px;
+              border-radius: 12px;
+          }
+           :host(.list-mode-item) .thumbnail-container {
+              width: 200px;
+              height: 120px;
+              flex-shrink: 0;
+              border-radius: 12px;
+          }
+          :host(.list-mode-item) .content {
+              flex-grow: 1;
+              text-align: left;
+              padding: 0 1rem;
+              overflow: hidden;
+          }
+          :host(.list-mode-item) .actions {
+              margin-left: auto;
+              padding: 1em;
+              flex-shrink: 0;
+          }
+
+          :host(.grid-mode-item) .video-item {
+              display: flex;
+              flex-direction: column;
+              align-items: stretch;
+              height: 100%;
+              overflow: hidden;
+              background-color: #262626;
               border-radius: 8px;
-              background-color:#262626;
-              color: white;
-              margin-bottom: 1rem;
+          }
+          :host(.grid-mode-item) .thumbnail-container {
               width: 100%;
+              aspect-ratio: 16 / 9;
+              height: auto;
+              width: auto;
+              flex-shrink: 0;
+              border-radius: 8px 8px 0 0;
+          }
+           :host(.grid-mode-item) .thumbnail-container img {
+                object-fit: cover;
+                border-radius: 8px 8px 0 0;
+           }
+          :host(.grid-mode-item) .content {
+              padding: 0.75rem 0.75rem 0.5rem 0.75rem;
+              text-align: left;
+              overflow: hidden;
+              flex-grow: 1;
+              display: flex;
+              flex-direction: column;
+          }
+           :host(.grid-mode-item) .content .name {
+               white-space: nowrap;
+               overflow: hidden;
+               text-overflow: ellipsis;
+               margin-bottom: 0.25rem;
+               font-size: 1.0em;
+           }
+           :host(.grid-mode-item) .content .description {
+               font-size: 0.85em;
+               color: #ccc;
+               display: -webkit-box;
+               -webkit-line-clamp: 2;
+               -webkit-box-orient: vertical;
+               overflow: hidden;
+               text-overflow: ellipsis;
+               line-height: 1.3;
+               margin-bottom: 0.5rem;
+               flex-grow: 1;
+           }
+          :host(.grid-mode-item) .actions {
+              padding: 0rem 0.75rem 0.75rem 0.75rem;
+              display: flex;
+              justify-content: flex-end;
+              margin-top: auto;
+              flex-shrink: 0;
+          }
+           :host(.grid-mode-item) .action-button {
+               padding: 0.4rem 0.6rem;
+               font-size: 0.9em;
```

Comment:
```suggestion
               font-size: 0.6em;
```
can we make this 0.6em? 0.9 looks big to me in grid view.

Reply:
Cool, changing